docs/Configuration.md

```diff
@@ -1739,14 +1739,14 @@ test('does not show prototypes for object and array inline', () => {
     array: [{hello: 'Danger'}],
   };
   expect(object).toMatchInlineSnapshot(`
-{
```

Comment:
jest agrees with this new indentation if regenerated, fwiw. possibly fixed back in https://github.com/jestjs/jest/pull/11560 or something